Rocket.Chat/en.lproj/Localizable.strings

```diff
@@ -28,6 +28,13 @@
 "alert.settings.set_mail_app.title" = "Oops!";
 "alert.settings.set_mail_app.message" = "No Mail Accounts. Please set up a Mail account in order to send email.";
 
+"alert.audio_message.success.send" = "Send";
+"alert.audio_message.success.title" = "Success";
+"alert.audio_message.success.message" = "You message was successfully recorded. Recording time: %@.";
+
+"alert.audio_message.error.title" = "Ooops!";
+"alert.audio_message.error.message" = "An error has occurred. Please, record your message again.";
+
```

Comment:
We've to add this strings to the other localizable files (in English is fine if you don't know the other languages). Is just to keep it in sync.